internal/storage/sql/testing/testing.go

```diff
@@ -43,6 +45,10 @@ func (d *Database) Shutdown(ctx context.Context) {
 		_ = d.Container.StopLogProducer()
 		_ = d.Container.Terminate(ctx)
 	}
+
+	if d.dbfile != nil {
+		os.Remove(*d.dbfile)
```

Comment:
my only nit here is that sometimes when debugging test failures its nice to be able to load the sqlite DB to see the current state of the DB. I like the idea of moving it to a temp file, but that does make it hard to find the file to open, also this automatic removing of the file makes it impossible to load the DB on failure.

we could potentially move this cleanup to the start method, so you could still load the db after tests failed, it would only be removed on the next run

Reply:
💯 I ended up here because I had some funny state in that test file. Which makes me think the initial teardown process is not perfect. I see I forgot to add `oplock` to this list:
https://github.com/flipt-io/flipt/blob/main/internal/storage/sql/testing/testing.go#L117

I suspect that might have something to do with it. Likewise, I should port the thing I did in `import` (use `Down` via the migrator here). So that we continue to test our Down operation and ensure we don't forget this stuff.

I like that the default is to not leave DB files around. I suppose putting it in `/tmp` at-least means the OS will clear it away eventually.

Other ideas could be:

(a) log the file name and add a flag to preserve it (`--test-preserve-database`), or
(b) let the test invoker explicitly supply the DB path. i.e. when `FLIPT_DB_URL` is set in the test env use it as the target database. Or we could use `FLIPT_TEST_DB_URL`. When set use that and not the driver env var or a random path.

Reply:
oh yeah I kind of like `FLIPT_TEST_DB_URL`, wyt?

Reply:
Nice yeah. Let me do that. And in that scenario we wont clear it. Just leave it untouched.